### ISSUE TITLE
Setting to delete files after successful upload

### ISSUE THREAD

**Issue by winter7621** [yahoo/express-busboy#16]
So after the file (or image) has been uploaded to the server and the server has done whatever it needs to do with it (e.g. save to database or store in AWS, etc), the NodeJS server should delete the files from the `tmp` directory (or wherever they were saved)

**Comment by davglass** [comment]
Yes, I have no way of really knowing that you have acted on the file and done what you wanted. I guess I could add a method that cleans the files up but you would still need to call it when your process is done.

**Comment by davglass** [comment]
Didn't mean to close it :)

**Comment by winter7621** [comment]
That's OK. And a method that could be called to delete the files after processing is done is not a bad idea... I find having to use cronjobs a bit unsatisfactory

**Comment by ashjha1** [comment]
It created in my public directory some unwanted folders 

like 

'3d6c3049-839b-40ce-9aa3-b76f08bf140b' -> file -> myfile


I am not sure how to remove this.

**Comment by Kequc** [comment]
I suppose a file could be sent to any post route in my application and it would then remain in the tmp directory eternally. Maybe the best behaviour would be to cleanup after the request by default. If I wanted to keep the file beyond the request I'd move it to a safe location.

**Comment by danosaure** [comment]
If you go with the default clean up, please allow an option to prevent it.

    cleanup: false

Thanks,
